OptimizelySDKCore/OptimizelySDKCoreTests/OPTLYValidationTest.m

```diff
@@ -45,8 +45,9 @@ - (void)testMethodIsValidAttributeValueReturnsTrueForValidData
     XCTAssertTrue([(NSObject *)@0.0 isValidAttributeValue]);
     XCTAssertTrue([(NSObject *)@"" isValidAttributeValue]);
     XCTAssertTrue([(NSObject *)@"test_value" isValidAttributeValue]);
-    XCTAssertTrue([(NSObject *)[NSNumber numberWithDouble:(pow(2, 53))] isValidAttributeValue]);
-    XCTAssertTrue([(NSObject *)[NSNumber numberWithDouble:(-pow(2, 53))] isValidAttributeValue]);
```

Comment:
Oops, sorry for not being clear, but we should keep your original `double`-based assertions too! They're very relevant. 🙂